utils/rule_dir_json.py

```diff
@@ -173,6 +176,11 @@ def handle_remediations(product_list, product_yamls, rule_obj):
     return rule_remediations, r_products
 
 
+def quiet_print(msg: str, quiet, file: TextIO):
```

Comment:
We can't use type hints in this script because we test the script on CentOS 7 with Python 2.